pkg/util/templates/templates.go

```diff
@@ -70,6 +76,29 @@ type multiWindowMultiBurnAlertRule struct {
 	burnRate    string
 }
 
+
+func alertThreshold(windowSize, percent, label, burnRate string) string {
+
+	rule := "1-(sum(sum_over_time(probe_success{" + label + "}[" + windowSize + "]))" +
+		"/ sum(count_over_time(probe_success{" + label + "}[" + windowSize + "])))" +
+		"> (" + burnRate + "*(1-" + percent + "))"
+
+	return rule
+}
+
+func sufficientProbes(windowSize, label string) string {
+	window, _ := prometheus.ParseDuration(windowSize)
+	window_duration := time.Duration(window)
+	mPeriod, _ := prometheus.ParseDuration(serviceMonitorPeriod)
+	mPeriod_duration := time.Duration(mPeriod) 
+	necessarySamplesInWindow := 0.5
```

Comment:
this sounds like a number of samples you expect, but it's only the percentage of sample needed. What do you think of moving the calculation from the Sprintf to this line? I think the readability of promrule could benefit from that.